jujugui/static/gui/src/app/components/unit-details/_unit-details.scss

```diff
@@ -0,0 +1,14 @@
+.unit-details {
+    &__properties {
+        padding: 10px 20px;
+
+        p {
```

Comment:
Can we target a class here please. All component styles should not target raw elements.